### PR TITLE
Add spec loading state and animation

### LINKED ISSUE #7700: Spec Loading State
User Story: When viewing the Test Runner as it’s loading a spec file, I want to see a loading animation, so that I know that the file is loading and that something is happening.

**Acceptance Criteria**
- User will see 5 progressively pulsating circles
- Below circles user will see Your tests are loading…
  - font family: Muli
  - font size: 20px
- The first circle has a background color of #56b790
- Second circle: #4a88cc
- Third circle: #b96162
- Fourth circle: #e3b065
- Fifth circle: #a9abad
- Animation plays over 2 seconds
  - Animation repeats until spec file loads

ezgif-6-aec2bec79132

### PR DESCRIPTION
**Pull request by panzarino**
- Closes #7700

### User facing changelog

Adds a loading state so that a user knows when their tests are loading, especially if it takes longer than usual

### How has the user experience changed?

New loading state:

ezgif-6-6aa0b47815bd

### PR Tasks

- [ ] Have tests been added/updated?
- [x] Has the original issue been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)?

### CODE REVIEW

File: packages/reporter/src/runnables/runnables.scss [hunk 1]
```diff
@@ -263,4 +263,65 @@
...
+
+  .runnable-loading {
+    .runnable-loading-animation {
+      display: flex;
+      margin: 3.5rem auto 1.5rem;
+      padding: 0 5px;
+      width: 160px;
+
+      div {
+        animation: scaling 1.65s linear infinite;
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:275]:
```suggestion
        animation: scaling 1.65s ease-in-out infinite;
```

Smooths out the animation a little.

File: packages/reporter/src/runnables/runnables.scss [hunk 2]
```diff
@@ -263,4 +263,65 @@
...
+        border-radius: 50%;
+        height: 40px;
+        margin: 0 -5px;
+        transform: scale(0.5);
+        width: 40px;
+      }
+
+      div:nth-child(1){
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:283]:
```suggestion
      div:nth-child(1) {
```

File: packages/reporter/src/runnables/runnables.scss [hunk 3]
```diff
@@ -263,4 +263,65 @@
...
+        animation-delay:0.1s;
+        background: #56b790;
+      }
+
+      div:nth-child(2){
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:288]:
```suggestion
      div:nth-child(2) {
```

File: packages/reporter/src/runnables/runnables.scss [hunk 4]
```diff
@@ -263,4 +263,65 @@
...
+        animation-delay:0.2s;
+        background: #4a88cc;
+      }
+
+      div:nth-child(3){
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:293]:
```suggestion
      div:nth-child(3) {
```

File: packages/reporter/src/runnables/runnables.scss [hunk 5]
```diff
@@ -263,4 +263,65 @@
...
+        animation-delay:0.3s;
+        background: #b96162;
+      }
+
+      div:nth-child(4){
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:298]:
```suggestion
      div:nth-child(4) {
```

File: packages/reporter/src/runnables/runnables.scss [hunk 6]
```diff
@@ -263,4 +263,65 @@
...
+        animation-delay:0.4s;
+        background: #e3b065;
+      }
+
+      div:nth-child(5){
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:303]:
```suggestion
      div:nth-child(5) {
```

File: packages/reporter/src/runnables/runnables.scss [hunk 7]
```diff
@@ -263,4 +263,65 @@
...
+        animation-delay:0.5s;
+        background: #a9abad;
+      }
+
+      @keyframes scaling{
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:308]:
```suggestion
      @keyframes scaling {
```

File: packages/reporter/src/runnables/runnables.scss [hunk 8]
```diff
@@ -263,4 +263,65 @@
...
+        0%, 20%, 80%, 100%{
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:309]:
```suggestion
        0%, 20%, 80%, 100% {
```

File: packages/reporter/src/runnables/runnables.scss [hunk 9]
```diff
@@ -263,4 +263,65 @@
...
+          opacity: 100%;
+          transform: scale(0.5);
+        }
+
+        50%{
```
Comment (chrisbreiding) [inline packages/reporter/src/runnables/runnables.scss:314]:
```suggestion
        50% {
```

**Review by chrisbreiding — CHANGES_REQUESTED** [review]
Looks great! Just some little nitpicks to take care of.

File: packages/reporter/src/runnables/runnables.scss (hunk 1 above)
Comment (panzarino) [reply packages/reporter/src/runnables/runnables.scss:275]: I had this originally but as I tried to mimic the spec I changed this to `linear`. However, the final version definitely does look better with it.

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**